### PR TITLE
add avatar and alert components

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
added css components for avatar and alert. please do check file structure and naming conventions.

### CODE REVIEW

**Review by humzahasan — APPROVED** [review]
Looks Good! Few extra spaces, can be removed when refactoring.

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
+.alert{
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    gap: 1.6rem;
+    padding: 1.5rem 2rem;
+    border-radius: 2px;
+    width: 100%;
+    max-width: 700px;
```
Comment (itsmebipul00) [inline components/alert/alert.css]: you could use rems here

File: index.html [hunk 2]
```diff
@@ -4,10 +4,126 @@
...
+                <img src="./assets/photos/stock-photo3.jpg"class="avatar-image" alt="">
+            </div>
+            <div class="avatar">
+                <img src="./assets/photos/stock-photo4.jpg"class="avatar-image" alt="">
+            </div>
+            <div class="avatar avatar-size-sm">
+                <img src="./assets/photos/stock-photo.jpg"class="avatar-image" alt="">
+            </div>
+            <div class="avatar avatar-size-xs">
+                <img src="./assets/photos/stock-photo2.jpg"class="avatar-image" alt="">
```
Comment (itsmebipul00) [inline index.html]: alts are empty

**Comment by itsmebipul00** [comment]
Great work!! Tiny refactors, anyways good to merge

File: components/alert/alert.css (hunk 1 above)
Comment (iRohitGaur) [inline components/alert/alert.css]: try to use rem

File: components/alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    color: var(--DARK-GREY);
+    border: none;
+    border-radius: 999px;
+    padding:2px;
+    background-color: transparent;
+}
+
+.alert-close:hover{
+    background-color: #00000015;
+}
```
Comment (archit7448) [inline components/alert/alert.css:81]: you can use root for  color  and you can  remove spacing

**Review by purvasharmadev — APPROVED** [review]
looks good to me. 👍

File: components/alert/alert.css (hunk 1 above)
Comment (bhushannaikaroskar) [reply components/alert/alert.css]: hey resolved the changes. I hope my color naming conventions are good. If you think they are ok please approve my changes.

Comment (bhushannaikaroskar) [reply components/alert/alert.css]: done with changes.

File: index.html (hunk 2 above)
Comment (bhushannaikaroskar) [reply index.html]: Yup anyways these documentation part but still done with changes. If everything is ok please approve.

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

**Review by developer-anand — APPROVED** [review]
Good to go.

**Review by zoef-yt — APPROVED** [review]
Good to merge

**Comment by itsmebipul00** [comment]
Good to go

**Review by archit7448 — APPROVED** [review]

**Pull request merged**